src/Ilios/WebBundle/Controller/IcsController.php

```diff
@@ -156,12 +159,13 @@ protected function getDescriptionForEvent(UserEvent $event)
         usort($sessionMaterials, $callback);
         usort($courseMaterials, $callback);
 
-        $courseMaterials = array_map(function (CourseLearningMaterialInterface $learningMaterial) {
-            return $this->getTextForLearningMaterial($learningMaterial->getLearningMaterial());
+        $courseMaterials = array_map(function (CourseLearningMaterialInterface $learningMaterial) use ($now) {
```

Comment:
I think the type hint for `CourseLearningMaterialInterface` is supposed to be `LearningMaterialRelationshipInterface` to match the session one below.

Reply:
noted, will change that.